typescript/src/shared/prompts.ts

```diff
@@ -86,3 +86,11 @@ It takes three arguments:
 - amount (int, optional): The amount to refund in cents.
 - reason (str, optional): The reason for the refund.
 `;
+
+export const searchDocumentationPrompt = `
+This tool will take in a question input, search and retrieve relevant Stripe documentation to answer the question.
```

Comment:
I wonder if it is worth mentioning that it will take a user query about integrating with Stripe in their application and retrieve relevant Stripe documentation

(i.e., being more specific that this tool is specifically for integrating with Stripe)